modules/op-release-notes-1-0.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assembly:
+//
+// * openshift_pipelines/op-release-notes.adoc
+
+[id="op-release-notes-1-0_{context}"]
+= Release Notes for Red Hat {op-title} Technology Preview 1.0
+
+[id="new-features-1-0_{context}"]
+== New features
+* {op-title} 1.0 is available on {product-title} 4.4 and newer versions on channels *ocp-4.4.0* and *preview*.
+* {op-title} 1.0 has been updated to use :
+** Tekton Pipelines 0.11.1
+** Tekton `tkn` CLI 0.9.0
+** Tekton Triggers 0.4.0
+** ClusterTasks based on Tekton Catalog 0.11
+
+
+[id="fixed-issues-1-0_{context}"]
+== Fixed issues
+* link:https://issues.redhat.com/browse/SRVKP-637[SRVKP-637] : Previously, the `DeploymentConfig` Task triggered a new deployment build even when an image build was already in progress. This caused the deployment in Pipeline to fail. With this fix, the `deploy task` command  is now replaced with the `oc rollout status` command which waits for the in-progress deployment to finish.
+* link:https://issues.redhat.com/browse/SRVKP-631[SRVKP-631]: Support for `APP_NAME` parameter is now added in Pipeline templates.
+* link:https://issues.redhat.com/browse/SRVKP-616[SRVKP-616]: Pipeline template for Java S2I failed to look up image in the registry. With this fix, the look up will now be performed using the existing image Pipeline Resources instead of user provided `IMAGE-NAME` parameter.
+* link:https://issues.redhat.com/browse/SRVKP-383[SRVKP-383]: All the Openshift Pipelines images are now based on the Red Hat Universal Base Images(UBI).
+
+[id="known-issues-1-4-0_{context}"]
+== Known issues
+* If you are upgrading from an older version of {op-title}, you will need to delete your existing deployments before upgrading to version 1.0.
```

Comment:
@siamaksade @vdemeester @nikhil-thomas did we already agree on what happens to the old operator source? Will it cause the loss of data? If yes, we should state it here.

Reply:
As discussed in the call, we will not make the loss of data explicit here.

Reply:
@siamaksade - I have copied your comment from the [google doc ](https://docs.google.com/document/d/1A4ZXXvxt4WN4b-W4oAqzSX1ZIO2iGba_Ai9VfYUUFew/edit?disco=AAAAJd5Izuk)( now obsolete) here: 
" should we also include a note on "breaking changes" and upgrade process from dev preview?"

@vdemeester - request you to pls provide inputs here. Thanks

Reply:
@nikhil-thomas - Adding Nikhil as per our discussion on the daily call on upgrade procedure from DP to TP, and breaking changes.

Reply:
Hum, that's a good point. I think it might cause "data" loss if we consider past running object (taskrun, …) loss. I wonder if it should appear on the release notes. My initial thought would be **yes**, it doesn't hurt to tell the user.

Reply:
I can add this info as part of the Installation procedure. Something like: 
1. "Upgrading the existing OpenShift Pipelines Operator, or
2. Upgrading the OpenShift Pipeline Operator from Dev Preview release to Technical Preview release

Reply:
As discussed with Nikhil - it may not be possible to have an upgrade procedure from DP to TP. @nikhil-thomas could you please confirm? Thans